--- a/third_party/dawn/src/tint/reader/spirv/namer.h
+++ b/third_party/dawn/src/tint/reader/spirv/namer.h
@@ -15,6 +15,7 @@
 #ifndef SRC_TINT_READER_SPIRV_NAMER_H_
 #define SRC_TINT_READER_SPIRV_NAMER_H_
 
+#include <cstdint>
 #include <string>
 #include <unordered_map>
 #include <vector>
--- a/net/third_party/quiche/src/quiche/quic/core/qpack/qpack_stream_sender_delegate.h
+++ b/net/third_party/quiche/src/quiche/quic/core/qpack/qpack_stream_sender_delegate.h
@@ -7,6 +7,7 @@
 
 #include "absl/strings/string_view.h"
 #include "quiche/quic/platform/api/quic_export.h"
+#include <cstdint>
 
 namespace quic {
 
--- a/net/third_party/quiche/src/quiche/quic/test_tools/qpack/qpack_test_utils.h
+++ b/net/third_party/quiche/src/quiche/quic/test_tools/qpack/qpack_test_utils.h
@@ -11,6 +11,7 @@
 #include "absl/strings/string_view.h"
 #include "quiche/quic/core/qpack/qpack_stream_sender_delegate.h"
 #include "quiche/quic/platform/api/quic_test.h"
+#include <cstdint>
 
 namespace quic {
 namespace test {
--- a/third_party/abseil-cpp/absl/strings/internal/damerau_levenshtein_distance.cc
+++ b/third_party/abseil-cpp/absl/strings/internal/damerau_levenshtein_distance.cc
@@ -16,6 +16,7 @@
 
 #include <algorithm>
 #include <array>
+#include <cstdint>
 #include <numeric>
 
 #include "absl/strings/string_view.h"
--- a/third_party/abseil-cpp/absl/strings/internal/damerau_levenshtein_distance.h
+++ b/third_party/abseil-cpp/absl/strings/internal/damerau_levenshtein_distance.h
@@ -15,6 +15,7 @@
 #ifndef ABSL_STRINGS_INTERNAL_DAMERAU_LEVENSHTEIN_DISTANCE_H_
 #define ABSL_STRINGS_INTERNAL_DAMERAU_LEVENSHTEIN_DISTANCE_H_
 
+#include <cstdint>
 #include <numeric>
 #include <vector>
 
